docset/winserver2025-ps/networkcontroller/Set-NetworkControllerMultisiteConfiguration.md

```diff
@@ -0,0 +1,292 @@
+Specify this parameter only if you run this cmdlet on a computer that is not part of the network controller cluster.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ConnectionUri
+Specifies the Uniform Resource Identifier (URI) of a Network Controller. 
+
+```yaml
+Type: System.Uri
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Credential
+Specifies a user credential that has permission to perform this action. The default is the current user. 
+Specify this parameter only if you run this cmdlet on a computer that is not part of the network controller cluster.
+
+```yaml
+Type: System.Management.Automation.PSCredential
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Etag
+{{ Fill Etag Description }}
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 5
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Force
+Forces the command to run without asking for user confirmation
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: 8
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -PassInnerException
+This thumbprint must also be provided in the ClientCertificateThumbprint parameter in the Install-NetworkController or Set-NetworkController cmdlet so that Network Controller can authorize this user.
+The thumbprint must be provided only if the network controller client authentication is X509 certificates. 
+Get-NetworkController retrieves that client authentication and authorization information.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Properties
+Specifies a site configuration for Multisite Peering. Site configuration comes as a NetworkControllerMultisiteProperties object. This object can be defined as new-object Microsoft.Windows.NetworkController.NetworkControllerMultisiteProperties. Here are the following properties that can be changed:
+- CertificateSubjectName
+- [[Sites] \<NetworkControllerSite\>]
+    - ResourceID/RESTIPAddress
+    - IsPrimary
+    - State
+    - DeploymentID
+    - APIVersion
+    - ConfigurationState
+    - [[Properties] \<NetworkControllerSiteProperties\>]
+        - RestIPAddress
+        - CertificateSubjectName
+        - EncodedCertificate
```

Comment:
Most of this information belongs in an example.

```suggestion
Specifies a site configuration for Multisite Peering. This parameter expects a
**NetworkControllerMultisiteProperties** object. Create a **NetworkControllerMultisiteProperties**
object containing the property values you want to set.
```

Talk about the settable properties in the example.